### PR TITLE
doh-proxy: 0.0.8 -> 0.0.9

### PR DESCRIPTION
**Pull request by jonringer**
###### Motivation for this change
Had a weird error while i was doing a nix-review of another pr.

Actions:
- bumped version
- enabled tests

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/development/python-modules/unittest-data-provider/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1gn2ka4vqpayx4cpbp8712agqjh3wdpk9smdxnp709ccc2v7zg46";
+  };
+
+  meta = with lib; {
+    description = "PHPUnit-like @dataprovider decorator for unittest";
+    homepage = https://github.com/yourlabs/unittest-data-provider;
+    license = licenses.free;
```
Comment (alyssais) [inline pkgs/development/python-modules/unittest-data-provider/default.nix]:
```suggestion
    license = licenses.mit;
```

https://github.com/yourlabs/unittest-data-provider/blob/891d9ea888fb9b1197c40391a0c5c8c91f23c3d3/setup.py#L19

**Review by alyssais — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/unittest-data-provider/default.nix (hunk 1 above)
Comment (jonringer) [reply pkgs/development/python-modules/unittest-data-provider/default.nix]: hmm, guess I overlooked that, used to seeing a LICENSE file

File: pkgs/servers/dns/doh-proxy/default.nix [hunk 2]
```diff
@@ -1,17 +1,35 @@
...
   src = python3Packages.fetchPypi {
     inherit pname version;
-    sha256 = "0mfl84mcklby6cnsw29kpcxj7mh1cx5yw6mjs4sidr1psyni7x6c";
+    sha256 = "1fxzxipzdvk75yrcr78mpdz8lwpisba67lk4jcwxdnkv6997dwfp";
   };
 
-  propagatedBuildInputs = with python3Packages;
-    [ aioh2 dnspython aiohttp-remotes pytestrunner flake8 ];
-  doCheck = false; # Trouble packaging unittest-data-provider
+  nativeBuildInputs = [ pytestrunner flake8];
```
Comment (alyssais) [inline pkgs/servers/dns/doh-proxy/default.nix]:
```suggestion
  nativeBuildInputs = [ pytestrunner flake8 ];
```

**Review by alyssais — CHANGES_REQUESTED** [review]
re license: yeah, upstream really should provide a LICENSE or COPYING file. Might be an idea to open an issue?

fwiw, we can’t just assume software is free because it’s public. There has to be some indication of a free license.

File: pkgs/servers/dns/doh-proxy/default.nix (hunk 2 above)
Comment (jonringer) [reply pkgs/servers/dns/doh-proxy/default.nix]: oops, looks like i fogot to revisit this PR. Next time :)

**Pull request merged**